package.json

```
@@ -77,8 +77,8 @@
     "vue-observe-visibility": "^1.0.0",
     "vue-router": "^3.6.5",
     "vuex": "^3.6.2",
-    "youtubei.js": "^2.8.0",
-    "yt-channel-info": "^3.2.1"
+    "youtubei.js": "^2.9.0",
```

I can't see any code in this pull request, that uses any of the new changes in YouTube.js 2.9.0, so it doesn't need to be upgraded as part of this PR.
```suggestion
    "youtubei.js": "^2.8.0",
```